supported-libraries-schema.json

```diff
@@ -0,0 +1,64 @@
+{
+  "$schema": "https://json-schema.org/draft/2019-09/schema",
+  "$id": "This will be populated with actual link once uploaded",
+  "title": "Schema for the supported libraries list JSON",
+  "type": "array",
+  "default": [],
+  "items": {
+    "type": "object",
+    "required": [
+      "artifact",
+      "minimal_version",
+      "metadata_location",
+      "support_level"
+    ],
+    "properties": {
+      "artifact": {
```

Comment:
I wonder whether we can (re)use some ID/format here that can be used to identify the artifact in both Maven and Gradle. Maybe @melix has an idea for this?

Reply:
Can't we just use `$groupid:$artifactId`?